CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 Please refer to [UPGRADING](UPGRADING.md) guide for upgrading to a major version.
 
+## 7.3.0 - 2021-03-21
+
+### Added
+
+- Support for DER and P12 certificates [#2413](https://github.com/guzzle/guzzle/pull/2413(
+- Support the cURL (http://) scheme for StreamHandler proxies [#2850](https://github.com/guzzle/guzzle/pull/2850)
+- Support for `guzzlehttp/psr7` [#2874](https://github.com/guzzle/guzzle/pull/2874)
```

Comment:
This should be omitted. I assume we don't want to allow 2.x in here until 2.0.0 stable is out. That could coincide with Guzzle 7.4.0.

Reply:
Nope. 

That is exactly what we do want. We want to allow people to install it and report bugs before 2.0 is released.

Reply:
Then we need to update Guzzle's composer.json to allow `^2.0@dev`.